src/utils/staking.js

```diff
@@ -13,6 +13,7 @@ const {
     }
 } =  nearApiJs
 
+const NEAR_PER_BYTE = new BN(process.env.REACT_APP_NEAR_PER_BYTE || '100000000000000000000')
```

Comment:
Should this be available from RPC?  How is it calculated on Profile?

Reply:
Was following @evgenykuzyakov examples from near-sdk-rs